frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -367,6 +367,8 @@ function ReaderPaging:onSwipe(_, ges)
             self:onPagingRel(-1)
         end
     else
+        -- update footer (time & battery)
+        UIManager:broadcastEvent(Event:new("UpdateFooter"))
```

Comment:
broadcastEvent is a heavy operation since it iterates through all widgets in the system, how about we just call `self.view.footer:updateFooter()` here?